administrator/components/com_admin/script.php

```diff
@@ -545,4 +546,189 @@ private function cleanJoomlaCache()
 		$model->setState('client_id', 1);
 		$model->clean();
 	}
+
+	private function installWorkflow()
+	{
+		if (!JComponentHelper::isEnabled('com_workflow'))
+		{
+			return;
+		}
+
+		// Load languages
+		JFactory::getLanguage()->load('com_workflow', JPATH_ADMINISTRATOR);
+
+		// Check if any workflow is installed previously
+		$db = JFactory::getDbo();
+		$query = $db->getQuery(true);
+
+		$query->select("*")
+			->from($db->qn("#__workflows"))
+			->where($db->qn("id") . '= 1');
```

Comment:
It's not good to test for ID=1, because the workflow could be deleted...

Reply:
Perhaps check if the tables exist?

Reply:
table creation is done by sql script prior to executing this